src/django/api/management/commands/ecsmanage.py

```diff
@@ -0,0 +1,170 @@
+import boto3
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Run a one-off management command on an ECS cluster.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-e', '--env',
+            type=str,
+            choices=['staging', 'production'],
+            default='staging',
+            help="Environment to run the task in (staging or production)"
+        )
+
+        parser.add_argument(
+            'cmd',
+            type=str,
+            nargs='+',
+            help="Command override for the ECS container (e.g. 'migrate')"
+        )
+
+    def handle(self, *args, **options):
+        """
+        Run the given command on the latest app CLI task definition and print
+        out a URL to view the status.
+        """
+        self.env = options['env'].title()
+        cmd = options['cmd']
+
+        self.ecs_client = boto3.client('ecs')
+        self.ec2_client = boto3.client('ec2')
+
+        task_def_arn = self.get_task_def()
+        security_group_id = self.get_security_group()
+        subnet_id = self.get_subnet()
+
+        task_id = self.run_task(task_def_arn,
+                                security_group_id,
+                                subnet_id,
+                                cmd)
+
+        url = 'https://console.aws.amazon.com/ecs/home?region=us-east-1#'
+        url += f'/clusters/ecs{self.env}Cluster/tasks/{task_id}/details'
+
+        print('Task started! View status here:')
```

Comment:
See [here](https://docs.djangoproject.com/en/2.1/howto/custom-management-commands/) for recommendations on how to use `self.{stdout,stderr}` to write output from a management command.

Reply:
Good catch, I forgot about `self.stdout`.